test/unit/sources/deviantart_test.rb

```diff
@@ -31,25 +29,24 @@ def setup
 
     context "The source for a download-disabled DeviantArt artwork page" do
       should "get the image url" do
-        @site = Sources::Site.new("https://noizave.deviantart.com/art/test-no-download-697415967")
+        @site = Sources::Strategies.find("https://noizave.deviantart.com/art/test-no-download-697415967")
         assert_equal(["https://img00.deviantart.net/56ee/i/2017/219/2/3/test__no_download_by_noizave-dbj81lr.jpg"], @site.image_urls)
       end
     end
 
     context "The source for a DeviantArt image url" do
       should "fetch the source data" do
-        @site = Sources::Site.new("https://pre00.deviantart.net/b5e6/th/pre/f/2016/265/3/5/legend_of_galactic_heroes_by_hideyoshi-daihpha.jpg")
+        @site = Sources::Strategies.find("https://pre00.deviantart.net/b5e6/th/pre/f/2016/265/3/5/legend_of_galactic_heroes_by_hideyoshi-daihpha.jpg")
 
         assert_equal("hideyoshi", @site.artist_name)
-        assert_equal("https://hideyoshi.deviantart.com", @site.profile_url)
-        assert_equal("https://orig00.deviantart.net/9e1f/f/2016/265/3/5/legend_of_galactic_heroes_by_hideyoshi-daihpha.jpg", @site.image_url)
+        assert_equal("https://www.deviantart.com/hideyoshi", @site.profile_url)
+        assert_equal("https://pre00.deviantart.net/b5e6/th/pre/f/2016/265/3/5/legend_of_galactic_heroes_by_hideyoshi-daihpha.jpg", @site.image_url)
```

Comment:
Here the `orig00.deviantart.net` url was the correct image url to use. `pre00.deviantart.net/b5e6/th/pre/` is a sample url.